addons/sale_project/models/sale_order.py

```diff
@@ -48,10 +48,18 @@ def _compute_show_project_and_task_button(self):
             ('order_id.state', 'not in', ['draft', 'sent']),
             ('product_id.detailed_type', '=', 'service'),
         ], aggregates=['order_id:array_agg'])[0][0]
+        get_create_project_button = self.env['sale.order.line']._read_group([
+            ('order_id', 'in', self.ids),
+            ('order_id.state', 'not in', ['draft', 'sent']),
+            ('product_id.detailed_type', '=', 'service'),
+        ], aggregates=['id:recordset'])[0][0]
+        is_create_project_button = True
+        for sale_order in get_create_project_button:
+            is_create_project_button = sale_order.product_template_id.service_policy in ['delivered_timesheet', 'delivered_milestones']
```

Comment:
is this all really needed ? the conditions in the ``get_create_project_button`` are already checked in the ``order.id in show_button_ids`` below thus here I think they are redundant. 
Same with what we are doing in line 56-58